### DISCUSSION TITLE
"km/h" is converted to "km&#x2F;h"

### DISCUSSION THREAD

**Discussion by riopedre** [zachowj/node-red-contrib-home-assistant-websocket#364] [Q&A] [answered]
Hello. I'm using next flow
![imagen](https://user-images.githubusercontent.com/20193450/111882543-839ba180-89b6-11eb-826b-12f2fa6f72ae.png)
In 'Render template' node I have next template:
`"Wind speed too high ({{ states("sensor.aemet_wind_speed") }} {{ state_attr("sensor.aemet_wind_speed", "unit_of_measurement") }})"`
I'm using a wind speed sensor and one of its attribute (unit_of_measurement) that converts to "km/h" message.
Next node calls a script that sends a Telegram message. It uses a 'message' property with the string to send. I'm using this expression in data section of node:
`{
    "message": "{{payload}}"
}`
This is the debug result:
![imagen](https://user-images.githubusercontent.com/20193450/111882868-31f41680-89b8-11eb-9805-68162fa52f96.png)
Like you can see, first node make the conversion OK, but the 'call service node' changes the '/' character for '\&#x2F;' and it appears bad in Telegram.
What am I making wrong? Or where could be the problem? Thanks!

**Comment by zachowj** [comment, marked as answer]
<https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/mustache-templates.html>

**Comment by riopedre** [comment]
Thanks a lot. I have read the documentation many times but sometimes the answer is in front of the eyes and we can't see it.
